examples/realtime/hpcp-chroma/index.html

```diff
@@ -16,8 +16,10 @@
     <script src="https://cdn.jsdelivr.net/npm/essentia.js@0.1.0/dist/essentia-wasm.web.js"></script>
     <script src="https://cdn.jsdelivr.net/npm/essentia.js@0.1.0/dist/essentia-wasm.web.js"></script>
     <script src="https://cdn.jsdelivr.net/npm/essentia.js@0.1.0/dist/essentia.js-extractor.js"></script>
```

Comment:
@jmarcosfer Just as a reminder, we need to bump the version to 0.1.1 once we push latest version. Also needs to remove local imports of `essentia.js-model` add-on and so. That is, while merging the final PR for 0.1.1 release from dev branch to master. No need to change in this PR.